### PR TITLE
Update US Bank

### PR DESCRIPTION
**Pull request by stevea1**
US Bank now offers SMS (issue #3405)

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -1017,10 +1017,10 @@ websites:
 
     - name: US Bank
       url: https://www.usbank.com/
-      twitter: usbank
-      facebook: usbank
       img: usbank.png
-      tfa: No
+      tfa: Yes
+      sms: Yes
+      doc: https://github.com/2factorauth/twofactorauth.git
```
Comment (mxxcon) [inline _data/banking.yml]: This is not a correct docs url..

**Review by mxxcon — CHANGES_REQUESTED** [review]

**Comment by stevea1** [comment]
Fixed...

**Review by brycx — CHANGES_REQUESTED** [review]
I checked that docs URL, and I see no described steps of how to setup 2FA for an account. Have I missed something @stevea1?

**Comment by stevea1** [comment]
@brycx, right you are. I was not able to find any publicly accessible TFA documentation, so I removed the doc link.

**Comment by bishtawi** [comment]
I am a US Bank customer and I could not find any setting on their website allowing me to enable 2factor. I think the one-time passcode support mentioned [here](https://www.usbank.com/online-security/mobile-security.html) might for their mobile app only (as in the same sentence they mention fingerprint and facial recognition)? I cannot confirm this as I do not have their app installed.

**Comment by bishtawi** [comment]
Nevermind, I was able to follow the directions in https://github.com/2factorauth/twofactorauth/pull/3477 and enable SMS 2factor.

**Comment by mxxcon** [comment]
Looks like this was fixed in #3477

**Pull request closed without merging**